styles/style.css

```diff
@@ -0,0 +1,84 @@
+@import url("./fonts.css");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  max-width: 1440px;
+  font-weight: 400;
+}
+
+.advantages {
+  color: #383838;
+  background: #ffffff;
+  height: 569px;
+  padding: 120px 104px;
+}
+
+.advantages-title {
+  text-align: center;
+  font-weight: 500;
+  font-size: 40px;
+}
+
+.highlight {
+  width: 100px;
+  height: 4px;
+  background: #f5bd41;
+  margin: 20px auto 80px;
+  border: 0;
+}
+
+.advantages-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.advantages-row div {
+  width: 296px;
+}
+
+.advantages img {
+  display: block;
+  margin-bottom: 55px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.advantages-text {
+  font-size: 24px;
+  text-align: center;
+}
+
+.footer {
+  color: #ffffff;
+  background-color: #10367c;
+  height: 625px;
+  padding: 120px 272px 120px 104px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+  font-size: 20px;
+}
+
+.row h3 {
+  font-weight: 500;
+}
+
+.col {
+  display: flex;
+  flex-direction: column;
+  row-gap: 24px;
```

Comment:
The position of the columns is a bit different from the design.
Also, it depends on the width of the content - changing the longest titles will change the position of the columns. You may make it more constant by defining some width for the column.